### ISSUE TITLE
Update to docs v2.2

### ISSUE THREAD

**Issue by robsontenorio** [nuwave/lighthouse-docs#45]
The lighthouse site shows on top v2.1.

https://lighthouse-php.netlify.com

**Comment by spawnia** [comment]
Hey @robsontenorio i am going to work on the docs a bit more before i tag v2.2, as they. It's coming up for sure!

**Comment by robsontenorio** [comment]
Feedback: I am a newcomer, but most of tricks about usage  just got from Github issues , not on Docs :(

**Comment by spawnia** [comment]
If you find out how to do something and it is not part of the docs, i invite you to create
a PR. It is really easy, you just have to write some markdown and put it into the files in the `/docs` directory.

**Comment by robsontenorio** [comment]
I will. Just waiting 2.2 docs release.

**Comment by robsontenorio** [comment]
@spawnia Current version 2.2 docs looks great! Why not publish it and improve it incrementally?

It would be useful for newcomers. The 2.1 docs miss a lot of new features already published on main LightHouse package v2.2

**Comment by spawnia** [comment]
The issue i have with that is that we already have some changes laying around that will be only in v2.3. I am trying to get the next branch to a point where it has all of the important stuff of 2.2 in it before we tag it.

It is quite painful to maintain multiple versions and backport additions, so i basically want them in a good state before release.

**Comment by spawnia** [comment]
Feel free to accelerate that process by adding on the the `/docs` folder, as soon as we feel that is good enough, we will tag it v2.2

**Comment by spawnia** [comment]
Stuff that i think needs to be done before tagging v2.2:

- #44 
- #42 
- Document @can, @complexity, @middleware and @group under a guide "Security" (not @auth or @security!)

**Comment by robsontenorio** [comment]
i really wanna contribute. But, this lack between docs and main package version is painful confusing. 

For next versions is there any strategy to keep docs updated with main package version? The best scenario is “just release a package version if we have docs done.

**Comment by spawnia** [comment]
Since @chrissm79 and me are using Lighthouse in our work projects and we have limited time, features come first.
